scanner/src/funTest/kotlin/storages/ClearlyDefinedStorageFunTest.kt

```diff
@@ -121,4 +122,110 @@ class ClearlyDefinedStorageFunTest : StringSpec({
             )
         }
     }
+
+    "Scan results for packages without a namespace should be read correctly" {
+        val id = Identifier("NPM::iobroker.eusec:0.9.9")
+        val result = storage.read(id)
+
+        result.shouldBeSuccess {
+            it shouldContain ScanResult(
```

Comment:
There is already a test which covers a complete scan result above.
This assertion is quite detailed. I wonder if it is asserting "too much". Maybe just something like:
```
result.shouldBeSuccess {
    it.find { it.provenance.resolvedRevision == "327b125548c9b806490085a2dacfdfc6e7776803" } shouldNot beNull()
}
```

Reply:
Done in a similar but slight more verbose way.